src/game_map.cpp

```diff
@@ -847,6 +834,28 @@ int Game_Map::GetTerrainTag(int x, int y) {
 		return 1;
 	}
 
+	// Terrain tag wraps on looping maps
+	if (Game_Map::LoopHorizontal()) {
```

Comment:
This LoopHorizontal/LoopVertical checks are unnecessary, RoundX and RoundY already do them.